pages/_app.js

```diff
@@ -121,6 +91,19 @@ export default function App({ Component, pageProps }) {
     return () => clearInterval(interval);
   }, [isActive3]);
 
+  useEffect(() => {
+    setTrys(trys);
+    if (checkFruits) {
+      tryText;
+      save();
+
+      setTrys(1);
```

Comment:
this is confusing. shouldn't the tries be set to 0 after winning? or is this a cheating slots machine?

Reply:
if you win, they tries will be saved in localStorage (records) "save", then as next render-step the tries will be reset to 1 (setTrys(1), maybe i will change the name to "current try"